forte/mrdsrg-spin-integrated/master_mrdsrg.h

```diff
@@ -265,6 +265,8 @@ class MASTER_DSRG : public DynamicCorrelationSolver {
 
     /// Initialize Fock matrix
     void init_fock();
+    /// Initialize Fock matrix for ASET computations
+    void build_custom_fock();
```

Comment:
That DSRG needs a function to be called specifically in the embedding case worries me, as a design decision. I would think Hamiltonian construction would be separate from DSRG.

Reply:
This is an important point. This hides the details of the ASET computation inside a class that should not be responsible for it. Shouldn’t Master_MRDSRG be passed the right Fock matrix directly by the embedding driver?

Reply:
I think the problem is that the DSRG code do not take the information I give, it will always construct them, both for the reference energy and Fock matrix, so I have to go into these codes and ask them not to do so.